specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2024-10-01/redisenterprise.json

```diff
@@ -0,0 +1,2145 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-10-01",
```

Comment:
>2024 [](http://example.com/codeflow?start=16&length=4)

 Based on the diff, i see accesspolicyAssignment CRUD has been removed in this version but available in 09-2024-preview. Is that intentional and would it cause breaking changes.

Reply:
That is intentional. The accessPolicyAssignment is still in preview and not ready for GA yet.
That is the reason, this PR is based on the previous stable version(2024-02-01) and added features which are ready for GA

Reply:
Got it. Thanks for the clarification.